packaging/scripts/after_install.sh

```diff
@@ -50,7 +52,7 @@ update_selinux() {
 
 patch_updates() {
 	# make sure we have the latest
-	[ -f "${CRON}" ] && rm -rf "${CRON}"
+	[ -f "${CRON}" ] && rm -fv "${CRON}"
```

Comment:
why verbose here?

Reply:
It's good to print what's actually happening. I usually use `-v` with `rm` as a default.